### PR TITLE
Update dynflow to 0.8.11 and foreman-tasks to 0.7.18 (DEB)

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

File: plugins/smart_proxy_dynflow/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-smart-proxy-dynflow (0.8.11-1) stable; urgency=low
```
Comment (domcleal) [inline plugins/smart_proxy_dynflow/changelog]: Did you mean to update this package?

File: plugins/smart_proxy_dynflow/dynflow.rb [hunk 2]
```diff
@@ -1 +1 @@
-gem 'smart_proxy_dynflow', '0.0.7'
+gem 'smart_proxy_dynflow', '0.8.11'
```
Comment (domcleal) [inline plugins/smart_proxy_dynflow/dynflow.rb]: ditto

File: plugins/smart_proxy_dynflow/changelog (hunk 1 above)
Comment (iNecas) [inline plugins/smart_proxy_dynflow/changelog]: Yes, I have two commits in the PR, one fore dynflow, one for foreman-tasks

Comment (domcleal) [inline plugins/smart_proxy_dynflow/changelog]: This package is for the smart_proxy_dynflow plugin, but that's version 0.0.7 or 0.1.x. This file isn't for dynflow itself - you updated that in `dependencies/*/dynflow`.

Comment (iNecas) [inline plugins/smart_proxy_dynflow/changelog]: Right, I'm a bit slower today :/ It should be better now. Thank for spotting this.

**Comment by mmoll** [comment]
is this also for lower Forrman versions?

**Comment by iNecas** [comment]
Ideally for 1.12 and 1.11

**Comment by iNecas** [comment]
we have those versions in RPM

**Comment by mmoll** [comment]
cherry-picked:
```
[deb/develop 28656c3] Update dynflow to 0.8.11
[deb/develop ba54fda] Update foreman-tasks to 0.7.18
[deb/1.12 b798663]    Update dynflow to 0.8.11
[deb/1.12 be68f53]    Update foreman-tasks to 0.7.18
[deb/1.11 0737fb3]    Update dynflow to 0.8.11
[deb/1.11 445584b]    Update foreman-tasks to 0.7.18
```

díky @iNecas!

**Pull request closed without merging**